src/services/testNumber.ts

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+
+import { InvokeCommand, LambdaClient } from '@aws-sdk/client-lambda';
+import logger from '../util/logger';
+import { dynamoDBClientConfig } from '../config';
+
+export enum NumberTypes {
+  ZNumber = 'z-number',
+  TrailerId = 'trailerId',
+  TNumber = 't-number',
+  SystemNumber = 'system-number',
+}
+
+const lambdaClient = new LambdaClient({ region: dynamoDBClientConfig.region });
+
+export const generateNewNumber = async (numberType: NumberTypes): Promise<string> => {
+  if (process.env.AWS_SAM_LOCAL) {
+    return '123';
+  }
+  const input = {
+    path: `/${numberType}`,
+    httpMethod: 'POST',
+    resource: `/${numberType}`,
+  };
+  const command = new InvokeCommand({
+    FunctionName: process.env.TEST_NUMBER_LAMBDA_NAME,
+    InvocationType: 'RequestResponse',
+    Payload: JSON.stringify(input),
+  });
+  try {
+    const response = await lambdaClient.send(command);
+    const bufferResponse = Buffer.from(response.Payload!).toString('utf-8');
```

Comment:
```suggestion
    const bufferResponse = Buffer.from(response.Payload ?? '').toString('utf-8');
```
makes github happy. And slightly safer